src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -876,7 +876,7 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
                 this.room.roomId,
                 EventType.GroupCallMemberPrefix,
                 newContent,
-                this.useLegacyMemberEvents ? localUserId : `${localUserId}_${localDeviceId}`,
+                legacy ? localUserId : `${localUserId}_${localDeviceId}`,
```

Comment:
I'll make another PR for prefixing the state key with `_`, because it's a subtle yet significant change that's unrelated to the main purpose of this PR.

Reply:
perfect even better.